sfputil/main.py

```diff
@@ -793,33 +803,62 @@ def eeprom_hexdump_sff8636(port, physical_port, page):
 
     return output
 
+
+def eeprom_dump_general(physical_port, page, overall_offset, size, page_offset, no_format=False):
+    """
+    Dump module EEPROM for given pages in hex format. This function is designed for SFF8472 only.
```

Comment:
Isn't this function called for non-SFF8472 modules too? Asking in reference to `This function is designed for SFF8472 only`